### ISSUE TITLE
force load chunks does not function in 1.5.0 or 1.5.1

### ISSUE THREAD

**Issue by JheregJAB** [AllTheMods/ATM-6#1003]
**Describe the bug**
The force load feature of FTB Chunks does not appear to be loading chunks. Applies to server installs of ATM6 1.5.0 and 1.5.1.

**To Reproduce**
*Create a server.
*Claim some chunks.
*Place something in the chunk that should continue to function. I used a simple wheat farm for testing.
*Force-load the chunk containing the wheat farm.
*Log off.
*Allow some time to elapse
*Log on, and observe that the wheat did not grow.

**Additional context**
I noticed this was happening while playing on 1.5.0. I then tried upgrading to 1.5.1, which did not solve the problem. Finally, I tried on a clean install of 1.5.1 to rule out issues with my world file. The problem persisted even in the clean install. All of this testing was done in a server environment. I did not test single player.

**Comment by WhitePhant0m** [comment]
Question1: is `patch_chunkloading = true` what you see in your `world/serverconfig/ftbchunks-server.toml` file
Question2: how long did you wait?
You could try using something more easily confirm-able, like a furnace

**Comment by JheregJAB** [comment]
Answer1: yes, I found this config value and verified it as one of my first steps. `patch_chunkloading = true`. Sorry, should have mentioned that.

Answer2: For the first test, more than 3 hours. For subsequent tests, I enforced a timer of 15 minutes on myself. Sign out, wait 15 minutes, sign back in. When signing back in, the dirt had not even changed states from the dry state to the wet state, making it seem fairly obvious that the chunk had unloaded.

I'm about to sign on and play a bit for this evening, so I'll double check with a furnace as well and write back in a few minutes.

**Comment by JheregJAB** [comment]
ok, testing with a furnace seems to have worked as expected, so maybe its just a problem with `patch_chunkloading`.

Sign in, place furnace. place a cobble stack in it with some coal, immediately sign out. I then signed in 15 minutes later, and the furnace had completed. This would seem to suggest that the furnace ran. my crops (in the same chunk, btw) still did not advance.

**Comment by WhitePhant0m** [comment]
You might want to report his to FTB Chunks issue page then, since it would be an issue on their end

**Comment by GarthF** [comment]
TNP 3 had a similar issue. I believe it was traced back to Performant's 'fastChunkBlocks' setting.

**Comment by JheregJAB** [comment]
Thank you GarthF!

I disabled fastChunkBlocks, and on my first test (15 minutes) it appears that crops are now growing in force loaded chunks when the player is offline. I'm playing now, but I will also pay attention the next time I sign off for a protracted period and double check that this has definitely solved the issue.

**Comment by JheregJAB** [comment]
Ok, this time around I was signed out for more than an hour, and can confirm that my farms did continue to run.

For the sake of completeness, I'll link an issue I found on the tracker for TNP3.
https://github.com/The-Nexus-Project/Limitless-3/issues/81
It is not exactly the same, as it seems they didn't have ANY crops growing at all, but its close. The solution there is the same as here.

The solution to this issue, then, is to change the default config for the pack to set the 'fastChunkBlocks = false' in 'performant-common.toml'.

Related, I'd just like to mention that performant is closed source, and in researching this issue I found allegations that its changes are likely to cause issues with many other mods. The author has dismissed these conflicts in the past, but has also stated that performant is not a community project, but rather one he is working on for himself and simply allowing others to use. Given his sentiments, it is my personal opinion that performant may not be appropriate for a modpack such as ATM. That said, I am not aware of any other conflicts at this time, and don't feel strongly enough to advocate one way or the other beyond this ticket. I will link my sources for this information so that the relevant authorities may make their own decisions.
Sources:
https://github.com/someaddons/performant_issues/issues/70
https://github.com/someaddons/performant_issues/issues/59
